dot/utils_integration_test.go

```diff
@@ -35,7 +35,7 @@ func TestTrieSnapshot(t *testing.T) {
 	deepCopyTrie := tri.DeepCopy()
 
 	// Take Snapshot of the trie.
-	newTrie := tri.Snapshot()
+	snapshotedTrie := tri.Snapshot()
```

Comment:
revert this to `newTrie`; it's not really a `snapshotedTrie`, the `snapshotedTrie` is the trie on which the `Snapshot()` method is called.

Reply:
Updated.